pkg/app/piped/imagewatcher/watcher.go

```diff
@@ -35,7 +35,7 @@ import (
 )
 
 const (
-	defaultCommitMessageFormat = "Update image %s to %s defined at %s in %s"
+	defaultCommitMessageFormat = "Update %s in %s to %s"
```

Comment:
nits, I think the previous message format is more readable/useable than the new one since I could fulfill data to that template without checking for sample use 😅 
how about `Update image: %s in path: %s to registry: %s` 🤔

Reply:
@khanhtc1202 Thank you for your suggestion. To be honest I want to make this message as short and clear as possible. It actually has been already too long...
![image](https://user-images.githubusercontent.com/19730728/102747383-8c72dc00-43a3-11eb-8b39-76d4b4687ccf.png)

And this constant is used only for image watcher (not for piped operator). So how about just adding a comment on this line?

Reply:
it would be nice 👍 and it's still too long btw 🤣

Reply:
Yep, maybe I'll make a change on here again 😙

Reply:
okay, I will approve this PR, thank you 🙏